### PR TITLE
Update flag

### PR DESCRIPTION
**Pull request by ackagel**
Adds `--update` flag to `start_docker.sh` to address and close issue #166.  Readme has also been updated to provide user instructions for avoiding script overwrites.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,8 +53,28 @@ You can shut down the notebooks and close docker by entering control-c in the te
...
+bash start_docker.sh -u
+```
+
+to overwrite the old notebooks with the new updated versions.
+
+### WARNING
+
+If you have made changes to scripts that you would like to keep, rename them prior to running
+```
+bash start_docker.sh --update
```
Comment (ngreenwald) [inline README.md:77]: I think recopying the code here is confusing and makes them think they need to run it again. I would change this to something like: "prior to running the above command"

**Comment by ngreenwald** [comment]
I think we could do something a bit hacky, like how the jupyter notebooks are currently tested: https://stackoverflow.com/questions/13745648/running-bash-script-from-within-python. 

Basically, we can create a temp_dir with a couple fake .ipynb files, then make sure that with different flags passed in, we get them copied into a tempdir/scripts as appropriate.

**Comment by ackagel** [comment]
the template copying+logic can be moved into a separate script, which we can call from `start_docker.sh` and test through hacky python or BATS.  I think I'll try the python solution for now, since I've grown weary of bash and it's seven thousand different, critically incompatible platform variants (pls don't ask and bless yourself with the gift of bash-ignorance)

**Comment by ngreenwald** [comment]
That's a good point, this is the only thing that runs outside of the docker. What happens if someone has a windows setup?

**Comment by ackagel** [comment]
Technically, we only have instructions for bash users.  I think the assumption is that windows users are using git bash or the linux subsystem.  I'm not sure what flavor of bash those are...

Regardless, I'm pretty sure that the bash I've written is strictly gnu-bash (compatible with all variants) but I could double check (the main place where it might not be is `diff` which might have different exit codes and flags depending on the platform, but I'm p sure I handled the possible cases).

**Comment by ngreenwald** [comment]
Okay, in that case let's wait for someone to complain, and for now we'll
assume it's good.

**Pull request merged**